carbon-design-system/carbon-addons-iot-react
# feat(card): update breakpoint sizes

Closes #1826 

**Summary**

- Breakpoint sizes have been updated with the latest dashboard guild guidance
- CardEditForm now uses the currently selected breakpoint to determine the card size test

See https://ibm.ent.box.com/notes/718007286034?s=xcgyzxuwlws14mput7k8lq4ykz7ui1uu for the new breakpoints

**Change List (commits, features, bugs, etc)**

- Updated constants for breakpoints
- Updated TimeSeriesCard custom stroke and fill functions with jsdoc comments

**Acceptance Test (how to verify the PR)**

- Best place to test is in any of the DashboardEditor stories so thats its a bit more clear as to what the breakpoint is
- Add and select any card type
- Drag the window (or if you're in the breakpointSwitcher story, swap the breakpoint) and see that the card sizes reflect the new breakpoint guidance
- Also see that the card sizes are reflecting the new sizes based on the current breakpoint in the card size dropdown
